cedar-policy-core/src/entities.rs

```diff
@@ -1805,9 +1823,14 @@ mod json_parsing_tests {
         let eparser: EntityJsonParser<'_, '_> =
             EntityJsonParser::new(None, Extensions::all_available(), TCComputation::ComputeNow);
         assert_matches!(eparser.from_json_str(json), Err(e) => {
-            expect_err(json, &miette::Report::new(e), &ExpectedErrorMessageBuilder::error(
-                r#"error during entity deserialization: the key `bar` occurs two or more times in the same JSON object at line 11 column 25"# // TODO: put the line-column information in `Diagnostic::labels()` instead of printing it in the error message
-            ).build());
+            // TODO: put the line-column information in `Diagnostic::labels()` instead of printing it in the error message
```

Comment:
#599

Reply:
easier special case of that issue since we'd just need to convert line column back to a single index.